src/vs/workbench/contrib/extensions/test/electron-browser/extensionsViews.test.ts

```diff
@@ -124,7 +124,7 @@ suite('ExtensionsListView Tests', () => {
 		instantiationService.stubPromise(IExperimentService, 'getExperimentsByType', []);
 
 		instantiationService.stub(IExtensionService, {
-			getExtensions: () => {
+			getExtensions: (): any => { // TODO remove any
```

Comment:
Had to use `any` here as this function is returning a very incomplete type. Will open issue about removing `any`

Reply:
@mjbvz I can take care of this